lib/api/openid_connect/id_token.rb

```diff
@@ -65,6 +65,10 @@ def claims
       def build_sub
         Api::OpenidConnect::SubjectIdentifierCreator.create(@authorization)
       end
+
+      def root_url
+        Rails.application.routes.url_helpers.root_url(Rails.application.config.action_mailer.default_url_options)
```

Comment:
I don't like referring `Rails.application.config.action_mailer` here. I thought of setting `Rails.application.routes.default_url_options` globally for production environment (currently it is set only in tests). However I'm unsure if it's okay.